sdlib/d/gc/arena.d

```diff
@@ -516,6 +530,54 @@ private:
 	}
 }
 
+unittest allocSmallBulk {
```

Comment:
Where is the code that you are testing?

Reply:
But it really isn't this is just forwarding somewhere else. The logic tested is somewhere else, and so should the test be.